src/NHSD.BuyingCatalogue.Identity.UserDatabase/NHSD.BuyingCatalogue.Identity.UserDatabase.sqlproj

```diff
@@ -82,6 +82,30 @@
     <Build Include="Tables\Organisations.sql" />
   </ItemGroup>
   <ItemGroup>
+    <SqlCmdVariable Include="CREATE_EA_USER">
+      <DefaultValue>True</DefaultValue>
+      <Value>$(SqlCmdVar__5)</Value>
+    </SqlCmdVariable>
+    <SqlCmdVariable Include="EA_USER_FIRST_NAME">
+      <DefaultValue>Agency</DefaultValue>
+      <Value>$(SqlCmdVar__6)</Value>
+    </SqlCmdVariable>
+    <SqlCmdVariable Include="EA_USER_LAST_NAME">
+      <DefaultValue>User</DefaultValue>
+      <Value>$(SqlCmdVar__7)</Value>
+    </SqlCmdVariable>
+    <SqlCmdVariable Include="EA_USER_EMAIL">
+      <DefaultValue>user@agency.com</DefaultValue>
+      <Value>$(SqlCmdVar__3)</Value>
+    </SqlCmdVariable>
+    <SqlCmdVariable Include="EA_USER_PASSWORD_HASH">
+      <DefaultValue>AQAAAAEAACcQAAAAEEOFkNNrFpKmDC2TBv2CP/dzxfnjdXk97RoqRlunE/CGs2tmFcewKZj4M/fITiP2tg==</DefaultValue>
```

Comment:
I assume this is our test password & not something to be concerned about within a public repository?

Reply:
Correct. This is our usual test password.